spec/spec_helper.rb

```diff
@@ -15,14 +15,13 @@
 # it.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
-require 'rails_helper'
-require 'rspec-rails'
+require 'support/factory_bot'
 
 RSpec.configure do |config|
-  config.include Requests::JsonHelpers, type: :api
```

Comment:
Why remove this?

Reply:
I think this came up when I redid `rails generate rspec:install`.